.travis.yml

```diff
@@ -99,7 +99,15 @@ script:
       echo "Skipping script build step in Coverity branch/job";
     else
       travis_retry travis_wait 50 \
-          mvn clean verify --projects janusgraph-${MODULE} ${ARGS};
+          mvn clean verify --projects janusgraph-${MODULE} -Pcoverage ${ARGS};
+    fi
+
+after_success:
+  # Upload test coverage reports to Codecov
+  - if [ "${COVERITY_SCAN_BRANCH}" == 1 ] || [ -v COVERITY_ONLY ]; then
+      echo "Skipping test coverage report upload in Coverity branch/job";
+    else
+      bash <(curl -s https://codecov.io/bash);
```

Comment:
Never use codecov before but is this the intended way (i.e: use curl) to upload test coverage to Codecov ?

Reply:
Yes, see https://docs.codecov.io/v4.3.6/docs/about-the-codecov-bash-uploader -- that command is literally on that web page.

Reply:
Ah, interesting. Thx!